hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java

```diff
@@ -58,13 +57,9 @@
 @Singleton
 public class FileAppendHandler implements TransactionHandler {
     private static final Logger logger = LogManager.getLogger(FileAppendHandler.class);
-    private HederaFileNumbers fileNumbers;
 
     @Inject
-    public FileAppendHandler(@NonNull final HederaFileNumbers fileNumbers) {
-        // Exists for injection
-        this.fileNumbers = fileNumbers;
-    }
+    public FileAppendHandler() {}
```

Comment:
To keep this consistent with other classes:
```
    @Inject
    public FileAppendHandler() {
        // Exists for injection
    }
```
Otherwise it won't be clear why we keep this constructor around

Reply:
done thanks